openslides/users/static/js/users/site.js

```diff
@@ -458,6 +475,62 @@ angular.module('OpenSlidesApp.users.site', [
             }
             $scope.sortColumn = column;
         };
+        // define custom search filter string
+        $scope.getFilterString = function (user) {
```

Comment:
I think not, because not all properties of user should be in the filterString (e.g. default_passwort). So we have to decide for every model, what should be in the string.